src/index.ts

```diff
@@ -183,14 +186,14 @@ export type {
   Props,
   FlowStyleProps,
   ShortGridStyles,
-} from './components/types';
-export * from './styles/types';
-export * from './styles/list';
-export * from './styles/index';
+} from './tasty/types';
+export * from './tasty/styles/types';
+export * from './tasty/styles/list';
+export * from './tasty/styles/index';
 
 export { ModalProvider } from '@react-aria/overlays';
 export * from './utils/react';
-export * from './styled';
+export * from './tasty';
```

Comment:
so we're not exporting deprecated `styled` anymore?

Reply:
We should do it here, yes.